doc/manual/manual.tex

```diff
@@ -1486,6 +1486,85 @@ \subsubsection{Free surface stabilization}
 
 An example of a simple model which uses a free surface may be found in Section \ref{sec:cookbooks-freesurface}.
 
+
+\subsection{Nullspace removal}
+
+The Stokes equation (\ref{eq:stokes-1}) only involves symmetric gradients of the velocity, and as such 
+the velocity is determined only up to rigid-body motions (that is to say, translations and rotations).
+For many simulations the boundary conditions will fully specify the velocity solution, but for some 
+combinations of geometries and boundary conditions the solution will still be underdetermined.
+In the language of linear algebra, the Stokes system may have a nullspace.
+
+Usually the user will be able to determine beforehand whether their problem has a nullspace.  For instance, 
+a model in a spherical shell geometry with free-slip boundary conditions at the top and bottom will 
+have a rigid-body rotation in its nullspace (but not translations, as the boundary conditions do not 
+allow flow through them).  That is to say, the solver may be able to come up with a solution to 
+the Stokes operator, but that solution plus an arbitrary rotation is also an equally valid solution.
+
+Another example is a model in a Cartesian box with periodic boundary conditions in the x-direction, 
+and free slip boundaries on the top and bottom. This setup has arbitrary translations along the x-axis 
+in its nullspace, so any solution plus an arbitrary x-translation is also a solution.
+
+A solution with some small power in these nullspace modes should not affect the physics of the simulation. 
+However, the timestepping of the model is based on evaluating the maximum velocities in the solution, 
+and having unnecessary motions can severely shorten the time steps that \aspect{} takes. 
+Furthermore, rigid body motions can make postprocessing calculations and visualization more 
+difficult to interpret.  
+
+\aspect{} allows the user to specify if their model has a nullspace. If so, any power in the nullspace 
+is calculated and removed from the solution after every timestep.
+There are two varieties of nullspace removal implemented: removing net linear/angular momentum, and 
+removing net translations/rotations. 
+
+For removing linear momentum we search for a constant velocity vector $\bf c$ such that 
+
+\begin{equation*}
+\int_\Omega \rho ({\bf u - c}) = 0
+\end{equation*}
+
+This may be solved by realizing that $\int_\Omega \rho {\bf u} = {\bf p}$, the linear momentum, and 
```

Comment:
I stumbled over this as well, but it actually does. You probably think that the verb is missing, but the sentence says "This may be solved by realizing that A equals B, the linear momentum, and C equals D, the total mass."